app/controllers/orders_controller.rb

```diff
@@ -32,14 +34,14 @@ def edit
   # POST /orders
   # POST /orders.json
   def create
-    @order = Order.new(order_params)
+    @order = current_user.orders.build(order_params)
```

Comment:
done-- was already implemeneted in ability.rb just removed line @order = current_user.orders.build(order_params)